packages/runtime-core/__tests__/apiWatch.spec.ts

```diff
@@ -176,6 +176,21 @@ describe('api: watch', () => {
     ])
   })
 
+  it('watching multiple sources: undefined initial values and immediate: true', async () => {
+    const a = ref()
+    const b = ref()
+    let called = false
+    watch(
+      [a, b],
+      () => {
+        called = true
```

Comment:
The test seems not testing the change?

Reply:
You're right , I'm only testing it's being called initially (which previously it wasn't) but I need to validate the arguments passed. 

That actually made me find a part my fix did not cover - oldValue was not undefined when it should be.